utilities/works-pdf/src/utils/transformEstimateData.js

```diff
@@ -45,9 +46,9 @@ const transformEstimateData = (lineItems, contract, measurement, allMeasurements
 
       // if isDeduction is true then subtract noOfunit from consumedQuantity and if isDeduction is false then add noOfunit to consumedQuantity
```

Comment:
Change this Java docs to the exact thing which you have implemented in the below if condition

Reply:
yes, changed